helix-term/src/commands.rs

```diff
@@ -659,36 +668,79 @@ fn goto_line_start(cx: &mut Context) {
     )
 }
 
+fn goto_first_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 0);
+}
+
+fn goto_second_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 1);
+}
+
+fn goto_third_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 2);
+}
+
+fn goto_fourth_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 3);
+}
+
+fn goto_fifth_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 4);
+}
+
+fn goto_sixth_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 5);
+}
+
+fn goto_seventh_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 6);
+}
+
+fn goto_eight_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 7);
+}
+
+fn goto_ninth_buffer(cx: &mut Context) {
+    goto_buffer_by_idx_impl(cx.editor, 8);
+}
```

Comment:
If this were a command, it should be implemented as a single command that uses `cx.count()` for the buffer number. I don't think that adding another command is necessary though: you can add the bindings you're after with just the typable command like so:

```toml
[keys.normal]
"C-1" = ":b 1"
"C-2" = ":b 2"
# ...
```

Reply:
This is much cleaner. But the only location I could (relevantly) find "keys.normal" was in `book/src/remapping.md`. Does that mean there shouldn't keybindings by default? If yes, I have to disagree, as this is such an essential feature. Maybe I am using something incorrectly?

How would I use `cx.count()`, or better, how is count "provided"?.

> Returns 1 if no explicit count was provided

I seem to always get 1, probably I'm not using it correctly.

Reply:
We have typable variants of the `pipe` commands, so there's precedence. You can have a typable version, and another version that utilizes `cx.count()`